src/main/docs/guide/dbc/jdbc/jdbcConfiguration.adoc

```diff
@@ -61,5 +61,22 @@ As seen in the configuration above you should also configure the dialect. Althou
 
 IMPORTANT: The dialect setting in configuration does *not* replace the need to ensure the correct dialect is set at the repository. If the dialect is H2 in configuration, the repository should have `@JdbcRepository(dialect = Dialect.H2)` / `@R2dbcRepository(dialect = Dialect.H2)`. Because repositories are computed at compile time, the configuration value is not known at that time.
 
+=== Connection client info tracing
+
+In order to trace SQL calls using `java.sql.Connection.setClientInfo(String, String)` method, you can
+annotate repository with @api:data.connection.annotation.ConnectionClientInfo[] annotation. If boolean field `enabled` is set to false, then Micronaut will not issue `setClientInfo` calls to the JDBC connection.
+Fields in given annotation `module` and `action` can set custom module and action. If omitted, then module value will default
+to the class name (usually Micronaut Data repository class) annotated with the given annotation and action to the name of the method
+being executed. There is also property `clientInfoAttributes` allowing to set arbitrary client info attributes to be set to the connection client info.
+Annotation can be used on class or method, customizing module or action.
+
+For Oracle database, these values will be set to the JDBC connection using `setClientInfo("OCSID.MODULE", module)` and `setClientInfo("OCSID.ACTION", action)` respectively.
+Additionally, if `${micronaut.application.name}` is set, then it will be used to call JDBC connection `setClientInfo("OCSID.CLIENTID", appName)`.
+Please make sure that if you supply `OCSID.MODULE` or `OCSID.ACTION` in `ConnectionClientInfo` property `clientInfoAttributes` these values will be ignored because
+`module` and `action` attributes are going to be mapped to  `OCSID.MODULE` and `OCSID.ACTION`.
```

Comment:
```suggestion
Note that if you supply the attributes `OCSID.MODULE` or `OCSID.ACTION` in the `clientInfoAttributes` member of the `@ConnectionClientInfo` annotation  these values will be ignored because
`module` and `action` attributes are already mapped to  `OCSID.MODULE` and `OCSID.ACTION`.
```